docs/src/pages/component-demos/toggle-button/ExclusiveToggleButton.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles, createStyleSheet } from 'material-ui/styles';
+import ToggleButton, { ToggleButtonOption } from 'material-ui/ToggleButton';
+import Icon from 'material-ui/Icon';
+import Paper from 'material-ui/Paper';
+import Typography from 'material-ui/Typography';
+
+const styleSheet = createStyleSheet(theme => ({
+  root: {
+    width: '100%',
+  },
+  paper: {
+    padding: theme.spacing.unit * 4,
+    'margin-top': theme.spacing.unit,
+  },
+}));
+
+function alterText(value, selected) {
+  const div = document.getElementById('dummyDiv2');
+  if (selected && div !== null) {
+    if (value === '1') {
+      div.style.textAlign = 'left';
+    } else if (value === '2') {
+      div.style.textAlign = 'center';
+    } else if (value === '3') {
+      div.style.textAlign = 'right';
+    }
+  } else if (div !== null) {
+    div.style.textAlign = 'initial';
+  }
+}
+
+function ExclusiveToggleButton(props) {
+  const classes = props.classes;
+
+  return (
+    <div className={classes.root}>
+      <ToggleButton exclusive className="toggle">
+        <ToggleButtonOption
+          key="1"
```

Comment:
What's the key is for? I don't think that you need it.

Reply:
A key has to be specified on the option because i utilize the map function within the ToggleButton component and so a unique key should be specified.